frontend/src/components/__tests__/Button.test.tsx

```diff
@@ -4,12 +4,14 @@ import Button from "../Button";
 
 it("Button snapshot testing", () => {
   const treePrimary = renderer
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
I see this used enough that I wonder if it should be in the eslintrc?

Reply:
In fixing the ESLint config for the frontend it identified a number of these warnings in the legacy code - I believe the rule should be enabled application-wide however I have disabled it in these instances to ideally be refactored in the future to include more robust typing.